spec.md

```diff
@@ -719,7 +797,7 @@ If the successful response includes a `state` of `failed` then the Platform
 MUST send a deprovision request to the Service Broker to prevent an orphan
```

Comment:
Not just deprovision; in the case of a binding, it should also attempt to unbind.

Reply:
Good spot, fixed!